Library/Homebrew/formula.rb

```diff
@@ -1364,6 +1364,18 @@ def self.clear_installed_formulae_cache
     @installed = nil
   end
 
+  def self.comparison
```

Comment:
Thanks for the thoughts! I've moved this to utils/comparer.rb as `Comparer.tap_and_name` and tested it.